runtime/mainnet/src/benchmarks.rs

```diff
@@ -42,3 +67,183 @@ frame_benchmarking::define_benchmarks!(
 	[pallet_nfts, Nfts]
 	[pallet_assets, TrustBackedAssets]
 );
+
+parameter_types! {
+	pub ExistentialDepositAsset: Option<Asset> = Some((
+		RelayLocation::get(),
+		ExistentialDeposit::get()
+	).into());
+	pub const AssetHubParaId: ParaId = ParaId::new(1000);
+}
+
+impl pallet_xcm::benchmarking::Config for Runtime {
+	type DeliveryHelper = (
+		polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
+			XcmConfig,
+			ExistentialDepositAsset,
+			PriceForSiblingDelivery,
+			AssetHubParaId,
+			ParachainSystem,
+		>,
+	);
+
+	fn reachable_dest() -> Option<Location> {
+		Some(Location::parent())
+	}
+
+	fn teleportable_asset_and_dest() -> Option<(Asset, Location)> {
+		// No assets can be teleported.
+		None
+	}
+
+	fn reserve_transferable_asset_and_dest() -> Option<(Asset, Location)> {
+		ParachainSystem::open_outbound_hrmp_channel_for_benchmarks_or_tests(AssetHubParaId::get());
+
+		let who = frame_benchmarking::whitelisted_caller();
+		// Give some multiple of the existential deposit.
+		let balance = ExistentialDeposit::get() * 10_000;
+		let _ =
+			<Balances as frame_support::traits::Currency<_>>::make_free_balance_be(&who, balance);
+		let ah_on_pop: AccountId = LocationToAccountId::convert_location(&AssetHub::get())?;
+		let _ = <Balances as frame_support::traits::Currency<_>>::make_free_balance_be(
+			&ah_on_pop, balance,
+		);
+
+		// We can do reserve transfers of relay native asset to AH.
+		Some((
+			Asset { fun: Fungible(ExistentialDeposit::get()), id: AssetId(Location::from(Parent)) },
+			ParentThen(Parachain(1000).into()).into(),
+		))
+	}
+
+	fn set_up_complex_asset_transfer(
+	) -> Option<(xcm::prelude::Assets, u32, Location, Box<dyn FnOnce()>)> {
+		ParachainSystem::open_outbound_hrmp_channel_for_benchmarks_or_tests(ParaId::from(
+			AssetHubParaId::get(),
+		));
+		// Pop can only reserve transfer DOT.
+		// This test needs to be adapted as the features grow.
```

Comment:
could you please add a comment on one of the XCM config unit tests?

Basically that way we have this flow:
1.  we change XCM to support other asset transfers 
2. We now have a failing unit test
3. We fix unit test and see comment that says: "update set_up_complex_asset_transfer ..."

Reply:
makes sense, thanks!

Reply:
[25a3bf4](https://github.com/r0gue-io/pop-node/pull/479/commits/25a3bf42739f1fdbddf37ce5cc808231316b897e)